lib/KTextTruncator.vue

```diff
@@ -1,19 +1,4 @@
 <template>
-
-  <!--
```

Comment:
It's fine to cleanup comments that are specific to IE11, but please make sure to read them carefully and do not remove comments that have nothing to do with IE11 logic. This applies to all other removed comments in this file.

Reply:
That's Good Catch!